packages/flutter_tools/.analysis_options

```diff
@@ -7,12 +7,16 @@
 analyzer:
   language:
     enableSuperMixins: true
-  strong-mode: false #TODO(pq): enable once strong-mode runs clean
+  strong-mode: true
   errors:
     # we allow overriding fields (if they use super, ideally...)
     strong_mode_invalid_field_override: ignore
     # we allow type narrowing
     strong_mode_invalid_method_override: ignore
+    strong_mode_static_type_error: ignore
+    strong_mode_down_cast_composite: ignore
+    type_argument_not_matching_bounds: ignore
```

Comment:
It is true that `type_argument_not_matching_bounds` is not related to strong mode. I suspect you want this rule enabled, but it might be expedient to ignore it in this CL and then re-enable it in a separate CL that also contains fixes for the rule.